data-prepper-plugins/peer-forwarder/src/main/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerForwarderUtils.java

```diff
@@ -0,0 +1,63 @@
+package com.amazon.dataprepper.plugins.processor.peerforwarder;
+
+import com.linecorp.armeria.internal.shaded.bouncycastle.util.encoders.Hex;
+import io.opentelemetry.proto.trace.v1.InstrumentationLibrarySpans;
+import io.opentelemetry.proto.trace.v1.ResourceSpans;
+import io.opentelemetry.proto.trace.v1.Span;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public final class PeerForwarderUtils {
+    public static boolean isAddressDefinedLocally(final String address) {
```

Comment:
nit: I agree this static method is used by the Peerforwarder but this method unlike the other ones does something totally different. Other methods contain core business logic while this one actually does a check. I would suggest to move this method to PeerForwarderConfig or PeerForwarder.

Reply:
Also add a java docs for this method.